src/components/MarkMapMarker.css

```diff
@@ -1,10 +1,12 @@
 .MarkMapMarker-dot {
-  width: 13px;
-  height: 13px;
+  width: 14px;
+  height: 14px;
   margin-left: -7px;
   margin-top: -7px;
-  border-radius: 30px;
+  border-radius: 7px;
   cursor: pointer;
-
-  background: url(../img/dots.png) no-repeat center center;
+  background: linear-gradient(45deg, rgba(251, 117, 127, 1), rgba(254, 86, 174, 1));
+  -webkit-box-shadow: 0px 0px 0px 4px rgba(255,255,255, 0.3);
```

Comment:
you can remove the prefixed versions so -webkit -moz etc. as they are automatically added (less code :)